admin/src/app/hotel.service.ts

```diff
@@ -23,8 +25,9 @@ export class HotelService {
     return this.http.get<Hotel>(`${this.apiUrl}api/hotels/${id}`);
   }
 
-  public getHotelCards(pageParameters: PageParameters): Observable<HotelCard[]> {
-    const httpParams = pageParameters.getHttpParams();
+  public getHotelCards(pageParameters: PageParameters, filterParameters: FilterParameters): Observable<HotelCard[]> {
+    const params = Object.assign(pageParameters, filterParameters) as Params;
```

Comment:
Is it expected to change `pageParameters`?

Reply:
Do not understand this comment